### DISCUSSION TITLE
Opening new multi-file diff view?

### DISCUSSION THREAD

**Discussion by themightychris** [microsoft/vscode-discussions#2398] [Extension Development QnA] [answered]
Is there any way for an extension to open the new built-in multi-file diff view as seen here?

<img width="1136" alt="Screenshot 2024-11-14 at 5 17 15 PM" src="https://github.com/user-attachments/assets/adad7b56-2005-424c-8ff5-ce7e7e374cf0">

Is there a particular URI syntax to `vscode.diff` that can achieve this or another command?

**Comment by tamuratak** [comment, marked as answer]
You can use the `vscode.changes` command, which was missing in the document. It has been added at microsoft/vscode-docs/issues/7790.
